apps/studio/components/interfaces/Home/SecurityStatus.tsx

```diff
@@ -1,26 +1,27 @@
 import { CheckCircle2, Loader2 } from 'lucide-react'
 import { useState } from 'react'
 import { Button, PopoverContent_Shadcn_, PopoverTrigger_Shadcn_, Popover_Shadcn_, cn } from 'ui'
-
 import { useProjectLintsQuery } from 'data/lint/lint-query'
 import { useSelectedProject } from 'hooks/misc/useSelectedProject'
-
 import Link from 'next/link'
 import { WarningIcon } from 'ui/src/components/StatusIcon'
 
 export const SecurityStatus = () => {
   const project = useSelectedProject()
   const [open, setOpen] = useState(false)
-  const { data, isLoading } = useProjectLintsQuery({
-    projectRef: project?.ref,
-  })
+  const { data, isLoading } = useProjectLintsQuery({ projectRef: project?.ref })
 
-  const securityLints = (data ?? []).filter(
-    (lint) =>
-      lint.categories.includes('SECURITY') && (lint.level === 'ERROR' || lint.level === 'WARN')
-  )
+  const securityLints = (data ?? []).filter((lint) => lint.categories.includes('SECURITY'))
+  const errorLints = securityLints.filter((lint) => lint.level === 'ERROR')
+  const warnLints = securityLints.filter((lint) => lint.level === 'WARN')
+  const noIssuesFound = errorLints.length === 0 && warnLints.length === 0
```

Comment:
question: security lints will never be `INFO` level?

Reply:
there are info level lints, but I figure they're not high prio enough to trigger these notifications